api/graphql/schema.graphql

```diff
@@ -1649,6 +1681,20 @@ type Mutation {
   deleteUserSkill(id: UUID! @whereKey): UserSkill
     @can(ability: "delete", find: "id")
     @delete
+
+  # Assessment mutations
+  createAssessmentStep(
```

Comment:
Hmm, right now this mutation lets you add the special step 1 and 2 steps if you want.  Since we're going to have special backend logic for those I think this mutation should reject them.

Reply:
Definitely! Good call, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8022/commits/808a1dadd660d8420e0f1d926c73025604dbff85

Reply:
Oh, added at the schema level.  I didn't think of that - I like it.